geomesa-hbase/geomesa-hbase-distributed-runtime-hbase2/pom.xml

```diff
@@ -79,7 +88,6 @@
                             <transformers>
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
                             </transformers>
-                            <finalName>geomesa-hbase-distributed-runtime_${scala.binary.version}-${project.version}</finalName>
```

Comment:
Does the finalName come from the artifactId?

Reply:
yeah i think final name is not needed